dynawo/sources/Models/Modelica/PreassembledModels/UFLS3Steps.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
In addition to the comments made, it would be really nice to have dynawo test cases that test all these protections in the nrt folder !